config/blazegraph/blazegraph.production.properties

```diff
@@ -0,0 +1,47 @@
+#
```

Comment:
What are these .properties files for? Can we not check them into the repo and get them on the production server through scp? Or are these necessary to have in the repo?